src/util/hasher.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_UTIL_HASHER_H
+#define BITCOIN_UTIL_HASHER_H
+
+#include <crypto/siphash.h>
+#include <primitives/transaction.h>
+#include <uint256.h>
+
```

Comment:
I think it would be great if there was a small comment here explaining what belongs into this file and what doesn't to help future contributors so they don't have to go back in git history to understand it.

Reply:
Will add if I have to push again.